lib/jira/issue.rb

```diff
@@ -0,0 +1,22 @@
+module Jira
+  class Issue
+    SUPPORTED_ATTRIBUTES = ["summary", "project"].map(&:freeze).freeze
+    SUPPORTED_ATTRIBUTE_NAMES = SUPPORTED_ATTRIBUTES.map {|attr| "'#{attr}'"}.join(', ')
+
+    attr_reader :fields
+
+    def initialize(raw)
+      @raw = raw
+      @fields = @raw.fetch("fields")
```

Comment:
`@raw` is only used here so unnecessary to assign as instance variable (or any plan to use this in the future?)